lib/dev_toolbar/app/views/dev_toolbar/erd/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Entity Relationship Diagram</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    
+    <style>
+      body {
```

Comment:
The styles shown here are cribbed from the `public/404.html` styles in a default Rails app